### ISSUE TITLE
A plot is replaced in jupyter when plotting multiple holoviews objects

### ISSUE THREAD

**Issue by spott** [holoviz/holoviews#2676]
When I'm plotting multiple holoviews objects, such as:

```
display(hv.Curve(np.sin(np.arange(100))))
display(hv.Curve(np.cos(np.arange(100))))
```

The second one replaces the first.  This hasn't always been the case (a recent update caused it).

I consider this a pretty serious bug.  As a data scientist, a frequent use-case for plots is spitting them out from inside loops to watch as things change while training or creating a couple of plots quickly that I want to compare.

Is there a workaround for this?

**Comment by philippjfr** [comment]
This was caused when we added support for JupyterLab and therefore started handling the rendering of plots ourselves. We copied the code to render plots from bokeh, which does not support ``display`` I think. The fix for holoviews isn't complex, the lines [here](https://github.com/ioam/holoviews/blob/master/holoviews/plotting/widgets/widgets.js#L571), should be changed to:

```javascript
var nchildren = toinsert.length;
toinsert[nchildren-1].children[0].innerHTML = output.data[HTML_MIME_TYPE];
toinsert[nchildren-1].children[1].textContent = output.data[JS_MIME_TYPE];
```

I can open a PR to fix it over the weekend and we'll have a bugfix at some point.
